### PR TITLE
OI-84: General layout

### PR DESCRIPTION
**Pull request by nvelychenko**

### CODE REVIEW

File: themes/openideal_theme/templates/layout/page.html.twig [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        {% endif %}
+    </div>
+</div>
+
+{% if page.footer %}
+    <footer class="site-footer">
+        {% block footer %}
+            <div class="{{ container }}">
+                {# @Todo: move to block? Add Logo. #}
+                <div class="text-align-center text-white">Powered by <span class="font-weight-bold">OpenideaL</span>
```
Comment (db-Matroskeen) [inline themes/openideal_theme/templates/layout/page.html.twig]: Text should be translatable.

File: themes/openideal_theme/templates/layout/page.html.twig [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                                {{ page.navigation }}
+                            </nav>
+                        {% endif %}
+                    </div>
+                    {% if page.user %}
+                        <div class="site-navigation--user-info">
+                            {{ page.user }}
+                        </div>
+                    {% endif %}
+                    {#                    {% if page.footer %}#}
```
Comment (db-Matroskeen) [inline themes/openideal_theme/templates/layout/page.html.twig]: Do we need those commented lines here?

File: themes/openideal_theme/.gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 node_modules/
+package-lock.json
```
Comment (db-Matroskeen) [inline themes/openideal_theme/.gitignore]: Why package-lock.json should be ignored?

Comment (nvelychenko) [reply themes/openideal_theme/.gitignore]: I removed it from .gitignore.

File: config/install/openideal_theme.settings.yml [hunk 4]
```diff
@@ -6,16 +6,16 @@ favicon:
   mimetype: image/vnd.microsoft.icon
   path: ''
   url: 'http://openideal.docker.localhost/themes/contrib/bootstrap/favicon.ico'
```
Comment (db-Matroskeen) [inline config/install/openideal_theme.settings.yml]: I think we have to fix the favicon URL.

**Review by db-Matroskeen — CHANGES_REQUESTED** [review]

**Pull request merged**